modules/multi-architecture-modify-machine-set.adoc

```diff
@@ -4,23 +4,24 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-modify-machine-set_{context}"]
+= Adding a multi-architecture compute machine set to your Azure cluster
 
-= Adding a multi-architecture compute machine set to your cluster
+To configure an Azure cluster with multi-architecture compute machines, you must create an Azure ARM64 or AMD64 compute machine set that uses the ARM64 or AMD64 boot image. This adds ARM64 or AMD64 compute nodes to your cluster.
```

Comment:
remove `This adds ARM64 or AMD64 compute nodes to your cluster.`

Reply:
Updated as per the suggestion.